src/commands/node.mjs

```diff
@@ -483,6 +496,36 @@ export class NodeCommand extends BaseCommand {
           })
         }
       },
+      {
+        title: 'Upgrade the network deployment to enable mirror node with the address book',
```

Comment:
'Upgrade the network deployment to enable mirror node with the address book' -> `Enable mirror node`

Reply:
updated